transformations/mix_transliteration/requirements.txt

```diff
@@ -0,0 +1,8 @@
+aaransia
+numpy==1.18.1
+pandas==1.0.1
+python-dateutil==2.8.1
+pytz==2019.3
+six==1.14.0
+unicode==2.7
+Unidecode==1.1.1
```

Comment:
Unidecode is also a GPL one..

Reply:
Sorry, it's an oversight from my end. The scripts only require 3aransia which is under Apache Licence. I will update the requirements.txt in a bit.